src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/UseSymbolComparerInCollectionsTests.cs

```diff
@@ -0,0 +1,618 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Test.Utilities;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpSecurityCodeFixVerifier<
+    Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.UseSymbolComparerInCollectionsAnalyzer,
+    Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.Fixers.UseSymbolComparerInCollectionsFix>;
+using VerifyVB = Test.Utilities.VisualBasicSecurityCodeFixVerifier<
+    Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.UseSymbolComparerInCollectionsAnalyzer,
+    Microsoft.CodeAnalysis.Analyzers.MetaAnalyzers.Fixers.UseSymbolComparerInCollectionsFix>;
+
+namespace Microsoft.CodeAnalysis.Analyzers.UnitTests.MetaAnalyzers
+{
+    public class UseSymbolComparerInCollectionsTests
+    {
+        private const string SymbolEqualityComparerStubCSharp =
```

Comment:
📝 This is no longer needed. The `ReferenceAssemblies` can be specified to test both Roslyn 2.x (no `SymbolEqualityComparer`) and Roslyn 3.x.